app/commands.py

```diff
@@ -543,11 +546,13 @@ async def request(ctx: Context) -> Optional[str]:
     if bmap.status != RankedStatus.Pending:
         return "Only pending maps may be requested for status change."
 
+    status = "ranked" if ctx.args[0] == "rank" or ctx.args[0] == "ranked" else "loved";
+    
     await app.state.services.database.execute(
         "INSERT INTO map_requests "
-        "(map_id, player_id, datetime, active) "
-        "VALUES (:map_id, :user_id, NOW(), 1)",
-        {"map_id": bmap.id, "user_id": ctx.player.id},
+        "(map_id, requested_status, comment, player_id, datetime, active) "
+        "VALUES (:map_id, :status, :comment, :user_id, NOW(), 1)",
+        {"map_id": bmap.id, "status": status, "comment": "".join(ctx.args[1:]), "user_id": ctx.player.id},
```

Comment:
i think we should limit the length of the comment.

not sure if the message has already been truncated during packet handling, i think so? if i remember correctly, i truncate to ~2048 or so